README.md

```diff
@@ -14,25 +14,6 @@ docker-compose pull
 docker-compose up --remove-orphans
 ```
 
-To use the polkadot js temp fix for more than 65k events
```

Comment:
Ahhh, that's the reason for custom availj image?
Did Polka guys bump max events in recent v5.2.3 release?

Reply:
Yep, that's why I made this whole PR in the first place, i monitored it since a long time. FINALLY